packages/mdc-top-app-bar/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@material/top-app-bar",
+  "version": "0.0.0",
+  "description": "The Material Components for the web top app bar component",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web.git"
+  },
+  "keywords": [
+    "material components",
+    "material design",
+    "top app bar",
+    "top-app-bar",
+    "topappbar"
+  ],
+  "dependencies": {
+    "@material/base": "^0.29.0",
+    "@material/elevation": "^0.28.0",
```

Comment:
I guess we're planning to use this later? It's not currently used, seemingly.

Reply:
Yeah, this is a remnant of my prototype.